vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -257,6 +256,24 @@ func (r RefCountsMap) Decr(vol string) (uint, error) {
 	}
 	return rc.count, nil
 }
+// check if volume with source as mount_source belongs to vsphere plugin
+func matchNameforVMDK(mount_source string) bool {
+	var managedPluginMountStart string = "/var/lib/docker/plugins/"
+
+	// if plugin is used as a service
+	if strings.HasPrefix(mount_source, mountRoot) {
+		return true
+	}
+	// if plugin is used as managed plugin
+	// managed plugin has mount source in format:
+	// '/var/lib/docker/plugins/{plugin uuid}/rootfs/mnt/vmdk/{volume name}'
+	if strings.HasPrefix(mount_source, managedPluginMountStart) && strings.Contains(mount_source, mountRoot) {
```

Comment:
Yes, till PhotonOS upgrades to Docker 1.13 or later. Not sure that the plugin is used with Photon Controller yet.  I'll make a separate issue to support the refcounter with Photon once this change is submitted.